homeassistant/components/flux/switch.py

```diff
@@ -54,6 +56,11 @@
 CONF_BRIGHTNESS = "brightness"
 CONF_DISABLE_BRIGHTNESS_ADJUST = "disable_brightness_adjust"
 CONF_INTERVAL = "interval"
+CONF_INITIAL_STATE = "initial_state"
+
+INITIAL_STATE_ON = "state_on"
```

Comment:
Could we just start with a PR that just restores the state? Then once released, we can see if this config option is still needed.

Reply:
Ok, I'll do that

Reply:
@balloob Here you are :) I'll update #10515 immediately

Reply:
@balloob Done. I'll update home-assistant/home-assistant.io#10515 immediately